pkg/timestream/datasource_test.go

```diff
@@ -117,3 +127,35 @@ func TestCallResource(t *testing.T) {
 		})
 	}
 }
+
+func Test_runQuery_always_sends_db_name_with_quotes(t *testing.T) {
+	t.Run("db name provided without quotes runs query with quoted db name", func(t *testing.T) {
+		mockRunner := &fakeRunner{output: &timestreamquery.QueryOutput{Rows: []*timestreamquery.Row{}}}
+		ts := &timestreamDS{Runner: mockRunner}
+		sender := &fakeSender{}
+
+		assert.NoError(t, ts.CallResource(context.Background(), &backend.CallResourceRequest{
+			Method: "POST",
+			Path:   "tables",
+			Body:   []byte(`{"database":"db"}`),
+		}, sender))
+
+		require.Len(t, mockRunner.calls.runQuery, 1)
+		assert.Equal(t, &timestreamquery.QueryInput{QueryString: aws.String(`SHOW TABLES FROM "db"`)}, mockRunner.calls.runQuery[0])
+	})
+
+	t.Run("db name provided with quotes runs query with db name as-is", func(t *testing.T) {
```

Comment:
can you make these a table test? (you will need two similar ones for the other query type).

Reply:
I don't prefer the back-and-forth movement in order to read table tests... but going to try that out! Done.

However the names of the tests which I had before feel like too much... I like the explanation when I read them but right now it's cluttering. Any thoughts?

Reply:
yeah, since the title already explains the goal of the test (`Test_runQuery_always_sends_name_with_quotes`) you can skip the name, it can be inferred from the input-output